lib/text/ttml_text_parser.js

```diff
@@ -334,6 +342,10 @@ shaka.text.TtmlTextParser = class {
     const cue = new shaka.text.Cue(start, end, payload);
     cue.nestedCues = nestedCues;
 
+    if (cellResolution) {
+      cue.cellResolution = Object.assign({}, cellResolution);
```

Comment:
You don't need Object.assign.  The provided value won't be changed by the method that generated it, and you'll preserve type info in the compiler by assigning it directly.